test-case/check-capture.sh

```diff
@@ -91,6 +92,7 @@ do
                 dlogc "arecord -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q"
                 arecord -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q
                 if [[ $? -ne 0 ]]; then
+                    lsof $snd
```

Comment:
The user may be confused by the output of `lsof $snd`, because they don't know what the output actually is. my suggestion is: assign the result to a variable, if variable is empty, output nothing, because no device file is in use, and if variable is not empty, we output some hint info and the result. This is user friendly.
```
ret=$(lsof $snd)
[ -n $ret ] && dloge "Sound device file still in use:"
dloge $ret # maybe echo is better here?

```

Reply:
I think if none block it, so `lsof $snd` will dump nothing  
Here don't need to care about who use this device file  
your suggest is good, but I think it maybe complex

Reply:
My question is: if lsof dump some info, how can the user know what the info is? So I think the hint is important.